go/vt/vtgate/engine/ordered_aggregate.go

```diff
@@ -70,6 +69,10 @@ func (ap AggregateParams) isDistinct() bool {
 	return ap.Opcode == AggregateCountDistinct || ap.Opcode == AggregateSumDistinct
 }
 
+func (ap AggregateParams) changeField() bool {
```

Comment:
don't understand what check this is doing. could we add a comment to make it easier to understand?